packages/notifier/src/storesub.js

```diff
@@ -97,7 +103,7 @@ export const makeStoredSubscription = (
     serializeBodyFormat: 'smallcaps',
   }),
 ) => {
-  /** @type {Unserializer} */
+  /** @type {import('@agoric/internal/src/lib-chainStorage.js').Unserializer} */
```

Comment:
The `@import` on line 13 didn't work?

Reply:
shoot, just extra chars. not worth another push tho